test/codegen/end_to_end_test.dart

```diff
@@ -121,6 +121,10 @@ void main(List<String> args) {
     runQuery('K', new QueryOptions(
         includeInherited: true, withAnnotations: [lib.getType('AnnotC')]));
 
+    runQuery('J', new QueryOptions(includeMethods: true));
+    runQuery('J2', new QueryOptions(
```

Comment:
Thanks for giving it a shot! No problem, I opened #24 so we don't forget about it.